<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionFrame</class>
 <widget class="QFrame" name="QuestionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QTableView" name="tb_question">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>400</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_question">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>362</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
    </font>
   </property>
   <property name="text">
    <string>提问</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_page_pre">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>322</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_page_next">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>322</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_page">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
